api/commands/reminders/remind.js

```diff
@@ -59,23 +60,30 @@ module.exports = class RemindCommand extends Command {
       return msg.say(exceptions.past_time);
     }
 
-    let job = queue.create('reminder', {
-      target_id: target.id,
-      content: content
-    }).delay(millisecondsTillReminder).save(function(err) {
-      if (!err) {
-        return msg.direct(moment(chrono.parseDate(datetime)).calendar(
-          moment.now(), "M/D/YYYY h:mm a") + ', ' + target +
-          ' will be reminded "' + content + '" ');
-      }
-    });
+    return selectTz([target.username, target.discriminator])
+    .then(res => {
+      if (!res) {
+        return msg.say(exceptions.timezone_not_set);
+      } else {
+        let job = queue.create('reminder', {
+          target_id: target.id,
+          content: content
+        }).delay(millisecondsTillReminder).save(function(err) {
+          if (!err) {
+            return msg.direct(moment(chrono.parseDate(datetime)).calendar(

```

Comment:
trying to think of how to make this readable. how about something like -- I think this is an instance where a variable for readability purposes would be useful:
```
let scheduled_time = moment(chrono.parseDate(datetime)).
                       calendar(moment.now(), "M/D/YYYY h:mm a")
return msg.direct("${scheduled_time}, ${target} will be reminded ${content}");
```

thoughts?